buildenv/jenkins/README.md

```diff
@@ -92,6 +92,14 @@ You can also request a Pull Request build from the extensions repos or openj9-om
 - To trigger a Copyright Check
    - `Jenkins copyright check`
 
+- To trigger a SignedOffBy Check
+   - `Jenkins signed off by check`
+   - add a signedOffByCheck file
```

Comment:
Not sure what this means. It doesn't seem appropriate for the readme

Reply:
I guess we can remove these three lines. The section below has a description.